services/src/openkilda-gui/src/main/java/org/openkilda/controller/LoginController.java

```diff
@@ -70,6 +71,9 @@
 
     @Autowired
     private PermissionRepository permissionRepository;
+    
+    @Value("${twofa.application.name}")
+    private String twofaApplicationName;
```

Comment:
Remove twofa the property name should be application name only

Reply:
changes done